css/src/trojsten/protocol.less

```diff
@@ -0,0 +1,64 @@
+.protocol-set {
+    display: flex;
+    overflow-x: scroll;
+}
+
+.protocol-result {
+    font-weight: bold;
+}
+
+.protocol-time {
+    font-size: 0.8em;
+}
+
+.protocol-set-name {
+    font-size: 0.9em;
+}
+
+.protocol-set-result {
+    font-size: 1em;
+}
+
+@media (min-width: 992px) {
+    .protocol-set {
+        flex-wrap: wrap;
+        overflow-x: visible;
+    }
+}
+
+.protocol-box {
+    flex-shrink: 0;
+    width: 50px;
+    height: 50px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+
+    border: 1px solid #f5f5f5;
+    border: 1px solid #ddd;
+    background: #fff;
+}
+
+.protocol-ok {
+    background: #d6df75;
```

Comment:
Nechceme mat tieto veci vo variables? Pripadne pouzit existujuce bootstrap variables?

Reply:
Ano, chceme. Zabudol som to tam prepisat, ked som to kopiroval z htmlka.